TradeItIosTicketSDK2/TradeItPortfolioEquityPositionsTableViewCell.swift

```diff
@@ -38,4 +44,26 @@ class TradeItPortfolioEquityPositionsTableViewCell: UITableViewCell {
             self.chevron.image = UIImage(named: "chevron_down")
         }
     }
-   }
+    
+    @IBAction func buyButtonWasTapped(sender: AnyObject) {
+        let order = TradeItOrder()
+        order.action = TradeItOrderAction.Buy
+        order.linkedBrokerAccount = self.selectedPosition.linkedBrokerAccount
+        order.symbol = self.selectedPosition.position?.symbol
+        self.delegate?.buyButtonWasTappedWith(order: order)
+    }
+    
+    @IBAction func sellButtonWasTapped(sender: AnyObject) {
+        let order = TradeItOrder()
+        order.action = TradeItOrderAction.Sell
+        order.linkedBrokerAccount = self.selectedPosition.linkedBrokerAccount
+        order.symbol = self.selectedPosition.position?.symbol
+        self.delegate?.sellButtonWasTappedWith(order: order)
+    }
+    
+}
+
+protocol TradeItPortfolioPositionsTableViewCellDelegate: class {
```

Comment:
I think we only need one of these (`tradeButtonWasTapped(forOrder order: TradeItOrder)`) OR we move the order creation to the delegate (`buyButtonWasTapped(forPosition position: TradeItPortfolioPosition)`)